README.md

```diff
@@ -105,13 +105,39 @@ This will ensure all event handlers will be registered rather being overruled by
 
 | Property | Description | Type |
 | -------- | ----------- | ---- |
-| getReferenceProps | The merged attributes for the `reference` element  |  (userProps?: HTMLAttributes<Element>) => Record<string, unknown> |
-| getFloatingProps | The merged attributes for the `floating` element |  (userProps?: HTMLAttributes<Element>) => Record<string, unknown> |
-| getItemProps | The merged attributes for when dealing with a list inside the `floating` element. |  (userProps?: HTMLAttributes<Element> & ExtendedUserProps) => Record<string, unknown> |
+| getReferenceProps | The merged attributes for the `reference` element. | (userProps?: HTMLAttributes<Element>) => Record<string, unknown> |
+| getFloatingProps | The merged attributes for the `floating` element. | (userProps?: HTMLAttributes<Element>) => Record<string, unknown> |
+| getItemProps | The merged attributes for when dealing with a list inside the `floating` element. | (userProps?: HTMLAttributes<Element> & ExtendedUserProps) => Record<string, unknown> |
 
 ### useHover
 
-(tbd)
+#### Usage
+
+```html
+<script>
+	import { useFloating, useInteractions, useHover } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
```

Comment:
We're going to have a lot on this page, keep removing the spacing between `const`.